### ISSUE TITLE
E/ED 2.0.0 support

### ISSUE THREAD

**Issue by marcemira** [hhff/ember-infinity#81]
Hello, just trying to make it work with both ember and ember-data 2.0.0, but it won't. Console shows:
```
Uncaught Error: You attempted to schedule an action in a queue (afterRender) for a method that doesn't exist

dpmv.js:15891 DEPRECATION: A property of <dpm@view:-outlet::ember533> was modified inside the didInsertElement hook

11443 Uncaught Error: You attempted to schedule an action in a queue (afterRender) for a method that doesn't exist

Uncaught TypeError: Cannot read property '0' of undefined

Uncaught TypeError: Cannot read property 'off' of null
```

Thanks for the awesome work :)

**Comment by hhff** [comment]
@mkorfmann has got a PR up for this, see #79 

We should have it by monday!  Sit tight.

**Comment by hhff** [comment]
(you can point to his fork in the meantime)

**Comment by marcemira** [comment]
:+1: Thanks, sorry I saw that PR just 2 minutes ago (came here to close this issue :p)

**Comment by hhff** [comment]
Keep it open!  We'll close it when there's a fresh new version out.

**Comment by hhff** [comment]
closed with v0.1.0
